doc/docs/Python_Data_Analysis_Tutorial.md

```diff
@@ -111,9 +115,51 @@ unix% grep tmfreqs tri-rods.out > tri-rods.tm.dat
 unix% grep tefreqs tri-rods.out > tri-rods.te.dat
 ```
 
-The TM and TE bands are both plotted below against the "k index" column of the data, with the special k-points labelled. TM bands are shown in blue (filled circles) with the gaps shaded light blue, while TE bands are shown in red (hollow circles) with the gaps shaded light red.
+Alternatively, we can use the Python interface and pass the results (`tm_gaps`, `te_gaps`, `tm_freqs`, and `te_freqs` variables) to matplotlib functions:
 
-<center>![](images/tri-rods-bands.gif)</center>
+```py
+import matplotlib.pyplot as plt
+
+fig, ax = plt.subplots()
+x = [i for i in range(len(tm_freqs))]
```

Comment:
Why doesn't `x = range(len(tm_freqs))` work here?

Reply:
It does, that was just poor style from me. Thanks!